sonar-project.properties

```
@@ -3,7 +3,7 @@ sonar.organization=toucantoco
 
 # This is the name and version displayed in the SonarCloud UI.
 sonar.projectName=Toucan Connectors
-sonar.projectVersion=3.5.1
+sonar.projectVersion=3.5.1.1
```

```suggestion
sonar.projectVersion=3.5.2
```
3.5.1.1 is invalid semver